### DISCUSSION TITLE
keyball39/片方のキーボードが反応しません...

### DISCUSSION THREAD

**Discussion by daseu** [Yowkees/keyball#499] [Q&A]
お世話になります。

当方、初めての自作キーボード、電装系の知識がほぼない素人のため、
ご助力いただけますと幸いです。。。
keyball39、USB-C、右手トラックボールです。

下記のマニュアルを参考にさせていただき、
https://github.com/Yowkees/keyball/blob/main/keyball39/doc/rev1/buildguide_jp.md
トラックボール手前までの作業が完了しているところです。

ファームウェアの書き込みが完了し、入力のテストをしていたのですが、
トラックボール側の基盤からの入力ができません。（右手側すべて）

リセットボタンの情報はとどいているもようです、、（押下時に液晶が反応する）

以下、試してみたことになります。
左手側USBケーブル、TRRS接続にて入力　→　左手側◯、右手側ｘ
右手側USBケーブル、TRRS接続にて入力　→　左手側◯、右手側ｘ

恐れ入りますが、何卒、よろしくお願いいたします。

![Uploading github2.jpg…]()
![Uploading github.jpg…]()
![Uploading github1.jpg…]()

**Comment by Unikedama** [comment]
Proマイクロの刺さっている位置は合っていますか？

**Comment by daseu** [comment]
早々のご助言をいただきありがとうございます。
また、申し訳ありません、画像がうまく貼り付けられておりませんでした。

Proマイクロの位置確認をしましたが、合っていると思うのですがいかがでしょうか。

![promicro1](https://github.com/Yowkees/keyball/assets/99178425/80d1ae06-085a-4b14-a0ef-e4ee21e7b70f)
![promicro2](https://github.com/Yowkees/keyball/assets/99178425/da9119da-2f93-412c-8997-1c4c0d12b61a)
![github](https://github.com/Yowkees/keyball/assets/99178425/2951c10c-9b09-4a95-9342-a43eee103c3e)
![github1](https://github.com/Yowkees/keyball/assets/99178425/fc520f05-fa9e-4e78-bc86-488088c7978c)
![github2](https://github.com/Yowkees/keyball/assets/99178425/84cf328c-3049-4e83-925f-885ee1f93bba)

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

Unikedama様もご助言いただきましてありがとうございます。

添付していただいた写真を確認させていただいたところ、すべてかは把握しきれませんがほとんどのダイオードの向きが逆にはんだ付けされているように見えますがどうでしょうか。
お手数ですがビルドガイドを参考にしながら確認していただけますと幸いです。

**Comment by daseu** [comment]
ご確認をいただきありがとうございます。

ダイオードの向きが逆向きとのこと...。
お恥ずかしい限りです...。
あらためてビルドガイドを参考/確認をし修正したいと思います。

**Comment by daseu** [comment]
大変お世話になります。
先般ご指摘いただいた箇所を修正したところ、
おかげさまで大部分が動作するようになりました。誠にありがとうございました。


ロープロファイル部分というのでしょうか、
キーボード一番下のキーが全て反応していないようです。。
（Remapにてキー割当てを変えて試してみたのですが、反応がなく..）
お知恵を拝借できますと幸いでございます..。
![000](https://github.com/user-attachments/assets/48dbc49c-0c8f-470c-b598-cddb955db07b)
![005](https://github.com/user-attachments/assets/b0058e2f-23bd-4ca3-949e-33335c283d0e)
![010](https://github.com/user-attachments/assets/648c8595-035b-414f-be6e-6aec4d25a069)
